docs/usage/types.md

```diff
@@ -584,6 +585,28 @@ With proper ordering in an annotated `Union`, you can use this to parse types of
 : type method for constraining strs;
   see [Constrained Types](#constrained-types)
 
+### Exotic Types
+Pydantic also supports exotic types, which are described in detail below:
+
+#### ImportString
+On model instantiation, pointers will be evaluated and imported,
+(distantly related behavior to [PyObject](https://docs.python.org/3/c-api/structures.html#c.PyObject)).
```

Comment:
I'm not sure this line really helps with the explanation.

Reply:
Removed the link.